reconcile/terraform_resources.py

```diff
@@ -458,10 +474,16 @@ def filter_tf_namespaces(namespaces, account_name):
         if account_name is None:
             tf_namespaces.append(namespace_info)
             continue
-        tf_resources = namespace_info.get('terraformResources')
-        if not tf_resources:
+        tf_provider_resources = \
+            namespace_info.get('terraformProviderResources') or []
+        tf_resources = namespace_info.get('terraformResources') or []
+        if not (tf_provider_resources or tf_resources):
             tf_namespaces.append(namespace_info)
             continue
+        for provider_resource in tf_provider_resources:
+            if provider_resource['provisioner']['name'] == account_name:
+                tf_namespaces.append(namespace_info)
```

Comment:
Can this lead to `namespace_info` being appended twice? Once for `tf_profided_resources` and another one for `tf_resources`?